clevertap-core/src/main/java/com/clevertap/android/sdk/ManifestInfo.java

```diff
@@ -20,6 +20,8 @@ public class ManifestInfo {
 
     private static String spikyProxyDomain;
 
+    private static String handshakeDomain;
```

Comment:
There is no getter for this field. Does it not need to be used anywhere?

Reply:
fixed it, assigned to default instance